cql3/Cql.g

```diff
@@ -592,7 +596,7 @@ updateStatement returns [std::unique_ptr<raw::update_statement> expr]
       ( usingClause[attrs] )?
       K_SET columnOperation[operations] (',' columnOperation[operations])*
       K_WHERE wclause=whereClause
-      ( K_IF (K_EXISTS{ if_exists = true; } | conditions=updateConditions { cond_opt = std::move(conditions); } ))?
+      ( K_IF (K_EXISTS{ if_exists = true; } | conditions=whereClause { cond_opt = std::move(conditions); } ))?
```

Comment:
This makes sense but should not refer to whereClause, because it's an IF clause.